lib/timeduration.ts

```diff
@@ -122,8 +123,9 @@ export class TimeDuration {
     return JSBI.equal(this.totalNs, ZERO);
   }
 
-  round(increment: number, mode: Temporal.RoundingMode) {
-    if (increment === 1) return this;
+  round(incrementParam: JSBI | bigint, mode: Temporal.RoundingMode) {
+    const increment = ensureJSBI(incrementParam);
+    if (JSBI.equal(increment, ONE)) return this;
```

Comment:
Unfortunately I had to change the type of this parameter back to JSBI, because of a test262 test where a custom time zone reported a very long day length. I should be able to change this back to Number when custom time zones are removed.